wallet/src/de/schildbach/wallet/ui/dashpay/BottomNavFragment.kt

```diff
@@ -33,9 +37,28 @@ abstract class BottomNavFragment(@LayoutRes contentLayoutId: Int) : Fragment(con
         mainActivity.bottom_navigation.menu.findItem(navigationItemId)
     }
 
+    /*fun showNavigation(show: Boolean) {
+        val navParentView = mainActivity.bottom_navigation.parent.parent
+        if (navParentView is KeyboardResponsiveCoordinatorLayout) {
+            navParentView.forceHideViewToHide = !show
+        } else {
+            mainActivity.bottom_navigation.visibility = if (show) View.GONE else View.VISIBLE
+        }
+    }*/
+
```

Comment:
This can be removed

Reply:
Removed https://github.com/dashevo/dash-wallet/pull/598/commits/95e566386798a689329efecdb02b68b91d71480f